src/ServicesFactory.ts

```diff
@@ -0,0 +1,16 @@
+import QueryBuilderServices from '@/QueryBuilderServices';
+import FetchSearchEntityRepository from '@/data-access/FetchSearchEntityRepository';
+import StatsvMetricsCollector from '@/data-access/StatsvMetricsCollector';
+
+const services = new QueryBuilderServices();
```

Comment:
I like this ServiceFactory, thanks!

Reply:
I just copied the logic from mediawiki mwhahahaha